### DISCUSSION TITLE
taking the opportunity and firmly grasping it

### DISCUSSION THREAD

**Discussion by chloe-hodges06** [samtupy/nvgt#137] [General]
good evening all, I'm starting to take the opportunity and learn new stuff when it comes to development :) shout out to my bf alex chapman for helping me get to the point where I want to learn this kind of stuff

**Comment by alexoloopios** [comment]
That's awesome, you are actually wanting to learn more advanced stuff, in fact we will both be learning stuff as I have started to learn some of the concepts and stuff

**Reply by chloe-hodges06** [reply]
yeah, I have to admit that I'm proud of myself

**Comment by samtupy** [comment]
Hi,

First, congratulations on a new found drive to learn programming / game development!

Unfortunately, github discussions are generally not a good place for idle conversation of this nature, but are instead intended for developers of the project to focus on and talk about specific features, requests, areas of improvement, ongoing tasks etc. No harm done here and I can understand that you may have been practicing the use of the Github interface, however the nvgt forum or discord server are probably better places for such a topic.

Most importantly, anybody who watches the github repository gets an email for every single comment in every issue or discussion here by default, sort of like a mailing list.

[some best practices regarding Github discussions and conversations](https://docs.github.com/en/discussions/guides/best-practices-for-community-conversations-on-github)

Thank you for your understanding. Have fun in your quest for knowledge and feel free to ask questions in appropriate places!
